pkg/encoding/v1/encoding_test.go

```diff
@@ -205,6 +217,7 @@ path: "/admin"
 	}
 
 	for _, tt := range tests {
+		t.Log("Test case: ", tt.Name)
 		t.Run("", func(t *testing.T) {
```

Comment:
Should we add `t.Run(test.Name, func(t *testing.T)` besides/instead of `t.Log...` here? WDYT?